lib/core-classes/DeviceService.ts

```diff
@@ -234,7 +234,14 @@ export class DeviceService {
       return { ...measure, name };
     });
 
+  
+    const listMeasures = []; // contain name and type of each measure to keep this data in device element
```

Comment:
it's a list of MeasureName no ? we can type this array

Reply:
Done